### ISSUE TITLE
/usr/bin/hostess: No such file or directory, In Arch Linux with 1.8

### ISSUE THREAD

**Issue by arlllk** [arkane-systems/genie#19]
When I run _genie -v -i_ with _1.8_ I get this

```
genie: no bottle present.
genie: initializing bottle.
/bin/sh: /usr/bin/hostess: No such file or directory
/bin/sh: /usr/bin/hostess: No such file or directory
```

And when i try to run 
`genie -c "systemctl status --no-pager"`
I get

```
/sbin/runuser: unrecognized option '--no-pager'
Try 'runuser --help' for more information.
genie: running command failed; nsenter returned 1.
```
Seems like using options triggers the bug because when I use 
`genie -c "systemctl status"`
It works fine

**Comment by cerebrate** [comment]
On the former, I presume you installed genie from the releases page, this being Arch? In which case, did you remember to install hostess manually from here - https://github.com/cbednarski/hostess ?

On the latter, oops. That'll be fixed in next build.

**Comment by arlllk** [comment]
Sorry, didn't read all the README, my mistake

**Comment by arlllk** [comment]
It came back
```
[arllk@Arley-wsl ~]$ genie -c -v "systemctl --no-pager status"                                                                                           genie: outside bottle 62.
genie: running command 'systemctl --no-pager status '
/sbin/runuser: unrecognized option '--no-pager'
Try 'runuser --help' for more information.
genie: running command failed; nsenter returned 1.
```
```
[arllk@Arley-wsl ~]$ genie --version
1.11
```

**Comment by cerebrate** [comment]
Hopefully fixed in 1.12 without reintroducing the other problem.

**Comment by arlllk** [comment]
works fine :)
